add-ons/lookups/payfone-tcpa-compliance/payfone-tcpa-compliance.6.x.py

```diff
@@ -1,15 +1,16 @@
 # Download the Python helper library from twilio.com/docs/libraries/python
 from twilio.rest import Client
 
-
 # Get your Account SID and Auth Token from twilio.com/console
 account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 auth_token = "your_auth_token"
 client = Client(account_sid, auth_token)
 
 number = client.lookups \
-               .phone_numbers("+16502530000") \
-               .fetch(add_ons="payfone_tcpa_compliance",
-                      add_ons_data={"payfone_tcpa_compliance.RightPartyContactedDate": "20160101"})
+    .phone_numbers("+16502530000") \
+    .fetch(
+        add_ons="payfone_tcpa_compliance",
+        add_ons_data={
+            "payfone_tcpa_compliance.RightPartyContactedDate": "20160101"})
```

Comment:
We should tweak yapf to have closing brackets and parentheses to match the indentation.